official/resnet/estimator_cifar_benchmark.py

```diff
@@ -97,22 +102,36 @@ def unit_test(self):
     flags.FLAGS.model_dir = self._get_model_dir('resnet56_1_gpu')
     flags.FLAGS.resnet_size = 8
     flags.FLAGS.dtype = 'fp32'
+    flags.FLAGS.hooks = ['ExamplesPerSecondHook']
     self._run_and_report_benchmark()
 
   def _run_and_report_benchmark(self):
     start_time_sec = time.time()
     stats = cifar_main.run_cifar(flags.FLAGS)
     wall_time_sec = time.time() - start_time_sec
 
+    examples_per_sec_hook = None
+    for hook in stats['train_hooks']:
+      if isinstance(hook, hooks.ExamplesPerSecondHook):
+        examples_per_sec_hook = hook
+        break
+
+    eval_results = stats['eval_results']
+    extras = {}
+    extras['accuracy_top_1'] = self._json_description(
+        eval_results['accuracy'].item(),
+        priority=0)
+    extras['accuracy_top_5'] = self._json_description(
+        eval_results['accuracy_top_5'].item())
+    if examples_per_sec_hook:
+      exp_per_second_list = examples_per_sec_hook.current_examples_per_sec_list
+      exp_per_sec = sum(exp_per_second_list[1:]) / (len(exp_per_second_list)-1)
```

Comment:
Is this to remove the warmup steps? Maybe we can document it?

Looks like there is already some logic to skip warmup steps in the `ExamplesPerSecondHook `?

Reply:
It skips the first 10.  **I will go ahead and tweak** this to just average them all and leave a note that the hook already skips the first 10 anyway.  It is a bit sloppy as we do not have flags to control that piped through the estimator example and I do not see the point at the moment.

Reply:
FIXED, good catch and idea.

Reply:
Thanks! Looks good to me.